src/DateRangePicker/DateRangePicker.js

```diff
@@ -195,22 +195,19 @@ class DateRangePicker extends React.Component<Props, State> {
   }
 
   // hover range presets
-  getWeekHoverRange = (date: moment$Moment) => {
+  getWeekHoverRange = (date: dayjs.Dayjs) => {
     const { isoWeek } = this.props;
 
     if (isoWeek) {
       // set to the first day of this week according to ISO 8601, 12:00 am
-      return [date.clone().startOf('isoWeek'), date.clone().endOf('isoWeek')];
+      return [date.startOf('isoWeek'), date.endOf('isoWeek')];
```

Comment:
Dayjs don't have startOf('isoWeek') at the moment

Reply:
yes, they are discussing about this...
https://github.com/iamkun/dayjs/issues/261
https://github.com/iamkun/dayjs/issues/215

I do not want patch dayjs, do we know the start of week? (sunday or monday)